h2o-extensions/xgboost/src/test/java/hex/tree/xgboost/XGBoostTest.java

```diff
@@ -1526,6 +1528,86 @@ public void testMakeDataInfo() {
     }
   }
 
+  @Test
+  public void testScoreContributions() throws IOException, XGBoostError {
```

Comment:
I'm missing end-to-end test that proves the new parameter works. E.g. in Python.

Reply:
it is not exposed, internal at this point